templates/taxi/driver_list.html

```diff
@@ -7,6 +8,11 @@ <h1>
         Create
       </a>
     </h1>
+    <form method="get" action="" class="form-inline">
+      {{ driver_search_form|crispy }}
+      <input type="submit" value="Seach" class="btn btn-secondary">
+    </form>
+    <br>
```

Comment:
Create a separate template for search form and include it where you need to avoid code duplication